silo/tests/test.py

```diff
@@ -209,6 +210,43 @@ def test_silo_detail_import_done(self):
         self.assertNotContains(response, "<h4>Import process running</h4>")
 
 
+class OneDriveReadTest(TestCase):
+    fixtures = ['../fixtures/read_types.json']
```

Comment:
Don't use fixtures. If you wanna add a ReadType, use the factories.

Reply:
Ok, i thought it would be cleaner if we need more than one. But sure, i will use factories.